service/role.go

```diff
@@ -339,6 +355,29 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 	return rt.(*RoleToken), err
 }
 
+func (r *roleService) storeTokenCache(key string, cd *cacheData, expTimeDelta time.Time, expTime int64) {
+	oldTokenCacheData, ok := r.domainRoleCache.Get(key)
+	r.domainRoleCache.SetWithExpire(key, *cd, time.Unix(expTime, 0).Sub(expTimeDelta))
+	if ok {
+		r.memoryUsage.Add(roleCacheMemoryUsage(cd) - roleCacheMemoryUsage(&oldTokenCacheData))
+	} else {
+		tokenCacheSize := roleCacheMemoryUsage(cd)
+		r.memoryUsage.Add(tokenCacheSize + int64(len(key)))
```

Comment:
```suggestion
		r.memoryUsage.Add(roleCacheMemoryUsage(cd) + int64(len(key)))
```

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/aeb335258dd94a6d6fb00394bfceabb26ea9b018